cachito/workers/requests.py

```diff
@@ -16,10 +21,17 @@ def get_requests_session(auth=False):
     """
     config = get_worker_config()
     session = requests.Session()
-    if auth and config.cachito_auth_type == "kerberos":
-        session.auth = requests_kerberos.HTTPKerberosAuth(
-            mutual_authentication=requests_kerberos.OPTIONAL
-        )
+    if auth:
+        if config.cachito_auth_type == "kerberos":
+            session.auth = requests_kerberos.HTTPKerberosAuth(
+                mutual_authentication=requests_kerberos.OPTIONAL
+            )
+        elif config.cachito_auth_type == "cert":
+            try:
+                session.cert = config.cachito_auth_cert
+            except AttributeError:
```

Comment:
This isn't needed since you validate that it's always set in `validate_celery_config` right?

Reply:
Right. I added it there just to be safe, in case the validation changes or is skipped for some reason in the future. Removed!